apps/charterafrica/src/components/Entity/Entity.js

```diff
@@ -90,30 +126,50 @@ const Entity = React.forwardRef(function Entity(props, ref) {
             alignItems="flex-start"
             justifyContent="center"
             item
-            sx={{ p: 2 }}
             xs={12}
-            sm={8}
+            md={8}
+            gap={2.5}
           >
             <RichTypography
-              textAlign={{ xs: "center", sm: "left" }}
+              textAlign="left"
               color="neutral.dark"
-              variant="h2SemiBold"
+              variant="h2"
               sx={{ width: "100%" }}
             >
               {name}
             </RichTypography>
+            {role ? (
+              <RichTypography
+                textAlign="left"
+                color="neutral.dark"
+                sx={{ width: "100%" }}
+                variant="p4"
+              >
+                {role}
+              </RichTypography>
+            ) : null}
```

Comment:
```suggestion
            <RichTypography
              textAlign="left"
              color="neutral.dark"
              sx={{ width: "100%" }}
              variant="p4"
            >
              {role}
            </RichTypography>
```
`RichTypography` handles null/empty values just fine. No need for `?` check.